ui/src/notebooks/context/notebook.tsx

```diff
@@ -1,19 +1,24 @@
 import React, {FC, useState, useCallback} from 'react'
 import {PipeData} from 'src/notebooks'
+import {FromFluxResult} from '@influxdata/giraffe'
 
 export interface PipeMeta {
   title: string
   visible: boolean
+  loading: boolean
```

Comment:
nice nice, probs fine for now but makes me wonder if this should be `RemoteDataState` or something like it?

Reply:
should most def do that in here. oversight as i was running between things